src/utils/UserOperationLib.sol

```diff
@@ -12,6 +12,7 @@ import {UserOperation} from "account-abstraction/interfaces/UserOperation.sol";
 library UserOperationLib {
     address constant ENTRY_POINT_V06 = 0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789;
 
+    /// NOTE: Why do we define errors here? it looks like they are only used in one file, and they are not used here? 
```

Comment:
Leftover from when I was first writing these and was iterating for where to put different errors. Can move the Manager-specific ones over there, but `TargetNotAllowed`, `SelectorNotAllowed`, and `ValueNotAllowed` I expect to be shared for other permission contracts too.